data/drug_induced_liver_injury/meta.yaml

```diff
@@ -0,0 +1,51 @@
+---
+name: drug_induced_liver_injury
+description: |-
+    Drug-induced liver injury (DILI) is fatal liver disease caused by drugs
+    and it has been the single most frequent cause of safety-related drug marketing
+    withdrawals for the past 50 years (e.g. iproniazid, ticrynafen, benoxaprofen).
+    This dataset is aggregated from U.S. FDA 2019s National Center for Toxicological
+    Research.
+targets:
+    - id: liver_injury
+      description: whether it can cause liver injury (1) or not (0).
+      units:
+      type: boolean
+      names:
+          - liver injury
+          - drug induced liver injury
+          - fatal liver disease caused by drugs
+          - liver toxicity
```

Comment:
@MicPie @phalem 

While we can ask an LLM to rephrase our prompt, I think it'd be great if we could already now try to use consistent declination/phrasing. For instance, names that can easily be inserted in `Is <name> <identifier>`  (`or what molecules in the list are <name>?` In the case of `boolean` targets (see contribution guide).

Reply:
@kjappelbaum
I'm not sure what is better:
* Adding (`What molecules in the list are`) `causing a liver injury`, `causing a liver toxicity` or simply add `causing a` to the prompt.
* Or we change everything to (`What molecules in the list are`) `injuring the liver`,`toxic to the liver`, etc.